core/kong-gateway/kong.production.yml

```diff
@@ -63,6 +63,48 @@ services:
         config:
           allowed_iss: [ "http://172.17.0.1:7000/auth/realms/diva-kc-realm" ]
 
+  - name: "Service-Management"
+    url: "http://service-management:3004/"
+    routes:
+      - name: services-route
+        paths:
+          - "/services/"
+        protocols:
+          - "http"
+          - "https"
+        strip_path: true
+        plugins:
+          - name: jwt-keycloak
+            config:
+            allowed_iss: [ "http://172.17.0.1:7000/auth/realms/diva-kc-realm" ]
+
+      - name: services-route-images
+        methods:
+          - POST
+          - PUT
+          - DELETE
+          - OPTIONS
+        paths:
+          - "/serviceImages"
+        protocols:
+          - "http"
+          - "https"
+        strip_path: false
+        plugins:
+          - name: jwt-keycloak
+            config:
+              allowed_iss: [ "http://172.17.0.1:7000/auth/realms/diva-kc-realm" ]
+
+      - name: services-route-get-images
```

Comment:
Why are we missing the jwt plugin here? @setaman it is missing also for users and assets.

Reply:
as we don't have a CDN or something, the GET route for images should be open. Otherwise the browser native request (e.g. `<img src="URL">`) can not pass without authentication

Reply:
Okay. I don't know if this is OK. We need to double check. For now it is sufficient.